src/main/java/com/partycipate/services/UserService.java

```diff
@@ -0,0 +1,22 @@
+package com.partycipate.services;
+
+import com.partycipate.models.Survey;
+import com.partycipate.models.User;
+
+public class UserService {
+
+    public static User createUser(String name, String email, String password){
```

Comment:
for now it is ok to just parse the PW into the user object. Hopefully the hash of the PWs will be here sooner or later